tests/drivers/w1/w1_api/testcase.yaml

```diff
@@ -7,6 +7,10 @@ common:
   platform_allow:
     - nucleo_g0b1re
     - nrf52840dk/nrf52840
+    - max32655evkit/max32655/m4
+    - max32680evkit/max32680/m4
+    - max32690evkit/max32690/m4
+    - apard32690/max32690/m4
```

Comment:
Why are MAX32666 boards not in this list?

Reply:
Because I forget to add it.

Reply:
max32666evkit is still missing it seems.

Reply:
Thanks added.